components/list/list-item-generic-layout.js

```diff
@@ -26,8 +26,18 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 
 	static get properties() {
 		return {
+			/**
+			 * @ignore
+			 */
+			frontRow: { type: Boolean, attribute: 'front-row', reflect: true },
```

Comment:
I think this should be made private... with leading underscore `_frontRow`, so that consumers who may look at this code thing it's something they can/should try setting.

Alternatively, we could reflect `_dropdownShowing` and `_tooltipShowing` and set up CSS selectors that target both those, to avoid having to come up with a name for this property to describe its purpose.

Reply:
I abandoned this approach because I had already made a typo in the handlers trying to keep the logic straight. Also, it's 100% cool to put classes on slots, and we can target them very easily. 

The alt approach would have been to put a wrapper `div` around the slots, but I didn't want to overcomplicate the DOM unnecessarily. This means that there's further reason to not put focusable/actionable things in the `content` slot as well.

Reply:
I'm reflecting these two properties with an underscore to denote that they are "private".

Should work in all browsers now.